### DISCUSSION TITLE
Adguard Home block all vids from n-tv.de

### DISCUSSION THREAD

**Discussion by Stern1900** [AdguardTeam/AdGuardHome#4907] [General]
Adguard Home block all vids on https://www.n-tv.de/.

**Comment by ppfeufer** [comment]
AGH itself does not block anything, so it's probably one of your DNS Blacklists.

**Reply by Stern1900** [reply]
I have now noticed that the "AdAway Default Blocklist" list, which is pre-installed for selection, blocks the videos.
